adafruit_display_text/scrolling_label.py

```diff
@@ -26,36 +26,42 @@
 __version__ = "0.0.0-auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_Display_Text.git"
 
+try:
+    from typing import Union, Optional
+    from fontio import BuiltinFont
+    from adafruit_bitmap_font.bdf import BDF
+    from adafruit_bitmap_font.pcf import PCF
+except ImportError:
+    pass
+
 import time
 from adafruit_display_text import bitmap_label
 
 
 class ScrollingLabel(bitmap_label.Label):
-
-    """
-    ScrollingLabel - A fixed-width label that will scroll to the left
+    """ScrollingLabel - A fixed-width label that will scroll to the left
     in order to show the full text if it's larger than the fixed-width.
 
     :param font: The font to use for the label.
-    :param max_characters: The number of characters that sets the  fixed-width. Default is 10.
-    :param text: The full text to show in the label. If this is longer than
-     `max_characters` then the label will scroll to show everything.
-    :param animate_time: The number of seconds in between scrolling animation
+    :type font: ~BuiltinFont, ~BDF, or ~PCF
```

Comment:
This can also change to `fontio.FontProtocol`:

```suggestion
    :type: ~FontProtocol
```